django-backend/fecfiler/transactions/migrations/0002_copy_scha.py

```diff
@@ -1,37 +1,6 @@
 # Generated by Django 3.2.12 on 2022-12-13 22:39
 
 from django.db import migrations
```

Comment:
We _might_ want to leave a comment for later explaining why this is empty.  It's not essential, though.